### PR TITLE
Fix issue #446

### PR DESCRIPTION
**Pull request by sandro-elsweijer**
**_Describe your changes here:_**
Fixes issue #446 

**_All these boxes must be checked by the reviewers before merging the pull request:_**

As a reviewer please read through all the code lines and make sure that the code is fully understood, bug free, well-documented and well-structured.

#### General
- [x] The reviewer executed the new code features at least once and checked the results manually

- [x] The code follows the [t8code coding guidelines](https://github.com/holke/t8code/wiki/Coding-Guideline)
- [x] New source/header files are properly added to the Makefiles
- [x] The code is well documented
- [ ] All function declarations, structs/classes and their members have a proper doxygen documentation
- [x] All new algorithms and data structures are sufficiently optimal in terms of memory and runtime (If this should be merged, but there is still potential for optimization, create a new issue)

#### Tests
- [x] The code is covered in an existing or new test case using Google Test

#### Github action

- [x] The code compiles without warning in debugging and release mode, with and without MPI (this should be executed automatically in a github action)
- [x] All tests pass (in various configurations, this should be executed automatically in a github action)

  If the Pull request introduces code that is not covered by the github action (for example coupling with a new library):
  - [ ] Should this use case be added to the github action?
  - [ ] If not, does the specific use case compile and all tests pass (check manually)

#### Scripts and Wiki

- [ ] If a new directory with source-files is added, it must be covered by the `script/find_all_source_files.scp` to check the indentation of these files.
- [ ] New Datatypes are added to t8indent_custom_datatypes.txt
- [ ] If this PR introduces a new feature, it must be covered in an example/tutorial and a Wiki article.

#### Licence

- [x] The author added a BSD statement to `doc/` (or already has one)

### CODE REVIEW

File: src/t8_geometry/t8_geometry_implementations/t8_geometry_occ.cxx [hunk 1]
```diff
@@ -652,11 +632,13 @@ t8_geometry_occ::t8_geom_evaluate_occ_hex (t8_cmesh_t cmesh, t8_gloidx_t gtreeid
        * The edges are indexed so that all edges which satisfy i_edge % 4 == 0 
        * have to multiplied with the inversed (1 - ref_coord) 
        * coordinate. All edges which satisfy i_edge % 4 == 1 have to multiplied with one 
-       * inversed ref_coord and so forth... */
+       * inversed ref_coord and so forth...
+       * An exception are edge 5 and 6, which have to be switched */
```
Comment (lukasdreyer) [inline src/t8_geometry/t8_geometry_implementations/t8_geometry_occ.cxx:636]: Can you expand on that, do they have to be switched to match your approach of multiplying with `(1-ref_coord)` with the t8code enumeration?

**Review by lukasdreyer — CHANGES_REQUESTED** [review]
Please add a short comment, explaining why edge 5 and 6 need to be switched.

File: src/t8_geometry/t8_geometry_implementations/t8_geometry_occ.cxx (hunk 1 above)
Comment (sandro-elsweijer) [reply src/t8_geometry/t8_geometry_implementations/t8_geometry_occ.cxx]: I tried my best

**Review by lukasdreyer — APPROVED** [review]
Thank you!

**Pull request merged**